test/services/MeasureService.test.ts

```diff
@@ -152,6 +157,42 @@ describe('MeasureService', () => {
         });
     });
 
+    it('returns a Bundle including the root lib and Measure when root lib has dependencies and id passed through args', async () => {
+      await supertest(server.app)
+        .get('/4_0_1/Measure/testWithRootLibAndDeps/$package')
+        .expect(200)
+        .then(response => {
+          expect(response.body.resourceType).toEqual('Bundle');
+          expect(response.body.entry).toHaveLength(3);
+          expect(response.body.entry).toEqual(
+            expect.arrayContaining([
+              { resource: LIBRARY_WITH_DEPS },
+              { resource: LIBRARY_WITH_NO_DEPS },
+              { resource: MEASURE_WITH_ROOT_LIB_AND_DEPS }
+            ])
+          );
+        });
+    });
+
+    it('returns a Bundle including the root lib, Measure, and when root lib has dependencies and id passed through body', async () => {
```

Comment:
Same comment as above.. I think we want to change this to "root Lib and Measure" instead of "root lib, Measure, and"